src/core/qwindowkit_windows.h

```diff
@@ -125,6 +125,9 @@ namespace QWK {
         QString stringValue(QStringView subKey) const;
         QPair<DWORD, bool> dwordValue(QStringView subKey) const;
 
+        template<typename T>
+        const std::optional<T> value(QStringView subKey) const;
```

Comment:
I didn't mean the return value should be const, sorry for my misleading word. You can remove the const keyword from the return value